src/main/java/frc/robot/commands/Climb.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.drivetrains.commands.DriveTankWithPID;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+import java.util.function.Supplier;
+
+public class Climb extends CommandBase {
+
+    private static final RootNamespace namespace = new RootNamespace("climb");
+    private static final Supplier<Double> speed = namespace.addConstantDouble("speed", 0);
+    private static final Supplier<Double> tolerance = namespace.addConstantDouble("tolerance", 0);
+    private static final Supplier<Double> setpoint = namespace.addConstantDouble("setpoint", 0);
+
+    private final Drivetrain drivetrain;
+
+    private boolean startedClimbing;
+
+    public Climb(Drivetrain drivetrain) {
+        addRequirements(drivetrain);
+        this.drivetrain = drivetrain;
+    }
+
+    @Override
+    public void initialize() {
+        startedClimbing = false;
+    }
+
+    @Override
+    public void execute() {
+        double pitch = drivetrain.getNavxPitch();
+        if (!startedClimbing) {
+            drivetrain.arcadeDrive(speed.get(), 0);
+            startedClimbing = Math.abs(pitch) > tolerance.get();
+        }
+        if (startedClimbing) {
+            if (Math.abs(pitch) <= tolerance.get()) {
+                climbOnCenter().schedule();
```

Comment:
You don't call commands inside commands, it might get weird. Instead you play with its functions